Configuration/RequestMiddlewares.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+    'frontend' => [
+        'middleware-identifier' => [
```

Comment:
Consider specifying exactly where you would like the middleware to be placed, so it's not too early and not too late.